backend/src/test/java/ai/verta/modeldb/ExperimentRunTest.java

```diff
@@ -8228,4 +8232,15 @@ public void cloneExperimentRun() throws ModelDBException, NoSuchAlgorithmExcepti
 
     LOGGER.info("Clone experimentRun test stop................................");
   }
+
+  @Test
+  public void randomNameGeneration() {
+    LOGGER.info("Random name generation test start................................");
+    GenerateRandomNameRequest.Response response =
```

Comment:
@ad-47 This class is not included in the commit, I'm getting a compilation error.

Reply:
@coryverta there are proto changes (added on the description) https://github.com/VertaAI/modeldb/pull/1658 to compile a new endpoint. as per the practice, proto changes are on other PR to easy review and early merge. sorry for the confusion.

Reply:
👍